README.md

```diff
@@ -0,0 +1,58 @@
+# Introduction
+
+The `python-csm-api-client` package is a Python client library for accessing
+Cray System Management APIs.
+
+## Installation as a dependency
+
+`python-csm-api-client` can be installed as a dependency in your Python project:
+
+```
+export PIP_EXTRA_INDEX_URL="https://artifactory.algol60.net/artifactory/csm-python-modules/simple"
+pip install python-csm-api-client
+```
+
+## Local installation with Poetry
+
+This project uses `poetry` to manage dependencies and development environments.
```

Comment:
I feel like it could be worth mentioning that it can be installed with `pip` too, e.g. if I'm working in my existing SAT virtualenv and want to install this library, I can run:
```
$ pip install /path/to/python-csm-api-client/
```

FWIW, while it doesn't install all of poetry in this case it does install `poetry-core`. Also, the `-e|--editable` option does not work.

Reply:
Oh neat, I didn't realize `pip` had support for `pyproject.toml`-based projects.

For others, here's more info on it: [https://pip.pypa.io/en/stable/reference/build-system/pyproject-toml/](https://pip.pypa.io/en/stable/reference/build-system/pyproject-toml/)